securesystemslib/util.py

```diff
@@ -457,3 +459,40 @@ def digests_are_equal(digest1: str, digest2: str) -> bool:
             are_equal = False
 
     return are_equal
+
+
+def b64enc(data: bytes) -> str:
```

Comment:
Agreed, especially since we already have base64 utility functions in the public (?) `formats` module. Let's start a `_utils.py` or `_internal/utils.py` just with the b64enc and b64dec from this PR.

Reply:
Ping